static/js/ducks/gcnEvents.js

```diff
@@ -3,22 +3,23 @@ import messageHandler from "baselayer/MessageHandler";
 import * as API from "../API";
 import store from "../store";
 
-export const FETCH_RECENT_GCNEVENTS = "skyportal/FETCH_RECENT_GCNEVENTS";
-export const FETCH_RECENT_GCNEVENTS_OK = "skyportal/FETCH_RECENT_GCNEVENTS_OK";
+const FETCH_GCN_EVENTS = "skyportal/FETCH_GCN_EVENTS";
+const FETCH_GCN_EVENTS_OK = "skyportal/FETCH_GCN_EVENTS_OK";
 
-export const fetchRecentGcnEvents = () =>
-  API.GET("/api/internal/recent_gcn_events", FETCH_RECENT_GCNEVENTS);
+// eslint-disable-next-line import/prefer-default-export
+export const fetchGcnEvents = () =>
+  API.GET("/api/gcn/event_views", FETCH_GCN_EVENTS);
```

Comment:
I think this API route might be out-of-date? It's api/gcn_events right?

Reply:
yeah, thanks. updated